### DISCUSSION TITLE
tmp volumes

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#3003] [Questions] [answered]
since tmp volumes don't get reused, are they automatically removed? since I know the ones mailcow uses aren't

**Comment by szaimen** [comment, marked as answer]
Didnt you asked this before already? :)

Yes they should get removed: https://github.com/nextcloud/all-in-one/blob/f86032a7e4ccff2562da39da3ff892dfb1820c98/php/src/Docker/DockerActionManager.php#L148 v=true does this.

**Reply by Zoey2936** [reply]
didn't know docker supports that, does compose also support that?

**Reply by szaimen** [reply]
> does compose also support that?

I dont know honestly. Probably also docker-compose can do that yes.
